src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -227,7 +230,10 @@ export default class ClientMetricReport {
       source?: string;
     };
   } = {
-    roundTripTime: { transform: this.identityValue, type: SdkMetric.Type.VIDEO_SENT_RTT_MS },
+    roundTripTime: {
+      transform: this.secondsToMilliseconds,
```

Comment:
The `type` is named as `VIDEO_SENT_RTT_MS`, which means the metrics should be converted to millisecond based.